### PR TITLE
Update secondary nav to match designs

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Abstract secondary nav out to a component
- Hard-code `is-selected` style until routing is implemented
- Rename `<Nav>` to `<PrimaryNav>`
- Add external links from Vanilla so icons show in Primary Nav

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify everything is as expected

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/799

### CODE REVIEW

File: src/components/SecondaryNav/_secondary-nav.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.p-secondary-nav {
+  .p-list__link {
+    color: #111;
```
Comment (barrymcgee) [inline src/components/SecondaryNav/_secondary-nav.scss]: I'm aware this should be a Vanilla variable but they cannot currently be accessed. Issue raised: https://github.com/canonical-web-and-design/jaas-dashboard/issues/13

**Comment by barrymcgee** [comment]
@spencerbygraves Any ideas yet on how this secondary nav might look on small screens?

**Comment by spencerbygraves** [comment]
@barrymcgee no, not at the moment.

File: src/components/SecondaryNav/_secondary-nav.scss (hunk 1 above)
Comment (huwshimi) [reply src/components/SecondaryNav/_secondary-nav.scss]: In maas-ui and the old gui we import the Vanilla bits we need at the top of the individual files, in this case that would be `@import "node_modules/vanilla-framework/scss/settings_colors";`

Comment (barrymcgee) [reply src/components/SecondaryNav/_secondary-nav.scss]: I've submitted a PR here: https://github.com/canonical-web-and-design/jaas-dashboard/pull/23

Managed to simplify the import to `"@import "vanilla-framework/scss/vanilla";"` which can be used uniformly in all partials and allow all Vanilla entities to be referenced. As Vanilla is engineering so all patterns are wrapped in mixins, only what is referenced is included so no performance overhead.

**Pull request merged**